scheduler/src/cook/components.clj

```diff
@@ -557,6 +561,17 @@
                    (merge {:interval-seconds 300
                            :dru-scale 1.0}
                           rebalancer))
+                       
+     :optimizer (fnk [[:config {optimizer nil}]]
+                     ;; TODO include a datomic connection to config so we can write purchase decisions
```

Comment:
Do we plan to do this TODO as part of this PR?

Reply:
No, going to remove this todo since it no longer is immediately actionable (purchasing decisions)